### PR TITLE
wip: new(ci): added update rules index gha.

### PR DESCRIPTION
**Pull request by cappellinsamuele**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area registry

**What this PR does / why we need it**:

This PR introduces the GHA responsible for updating the [index](https://github.com/falcosecurity/falcoctl/blob/gh-pages/index.yaml) on [`resitry.yaml`](https://github.com/falcosecurity/rules/blob/main/registry.yaml) updates.
This task is currently accomplished by `update-rules-index` prow job.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

`NONE`

### CODE REVIEW

File: .github/workflows/update_index.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    steps:
+      - name: Checkout Rules
+        uses: actions/checkout@v3
+
+      - name: Git set-up
+        run: |
+          apt-get update && apt-get install -y --no-install-recommends git && apt-get clean
+
+      - name: Setup Golang
+        uses: actions/setup-go@v3
```
Comment (darryk10) [inline .github/workflows/update_index.yml:34]: there is a v4 version available for `actions/setup-go` if we can use the most updated one.

**Comment by leogr** [comment]
/remove-lifecycle stale

@cappellinsamuele @FedeDP, any progress on this?

**Comment by FedeDP** [comment]
Hey Leo, given that his PR is trying to push to falcoctl gh-pages branch from the rules repo, i think this is not possible (or it's really hard) to do with github actions.
Imho this can be closed; leaving this job on poiana is not that wrong after all!

**Pull request closed without merging**